src/main/java/it/unibo/alchemist/boundary/wormhole/implementation/Wormhole2D.java

```diff
@@ -271,12 +271,12 @@ public Position getEnvPoint(final Point viewPoint) {
      * @return env space point
      */
     protected final PointAdapter envPointFromView(final PointAdapter viewPoint) {
-        final Point vp = viewPoint.toPoint();
+        final Point2D.Double vp = new Point2D.Double(viewPoint.toPoint().x, viewPoint.toPoint().y);
         final AffineTransform t = calculateTransform();
         try {
             t.inverseTransform(vp, vp);
         } catch (final NoninvertibleTransformException e) {
-            L.error(e.getMessage());
+            L.error(e.getMessage() + " Is the simulation visible FOR REAL?");
```

Comment:
This is not OK. This logging is unclear and deletes the whole stacktrace. Plus, it does not conform to the SLF4J's way for quick and effective logging. Please change it to:
```java
L.error(SOME_INTELLIGIBLE_MESSAGE_THAT_EXPLAINS_WHAT_IS_HAPPENING, e)
```